README.md

```diff
@@ -207,6 +208,19 @@ Hellonext helps you close the user feedback loop. Comes with feedback portal, ro
 | Platform  | Web                                                                |
 | URL       | https://hellonext.co                                               |
 
+### Screeb
+
+Screeb is a Product-Led User Research: Product Analytics, In-App Survey, User Recruitment...
+
+![screeb](media/screeb.png)
+
+| Property  | Value                                                              |
+|-----------|--------------------------------------------------------------------|
+| Developer | [Screeb, Inc.](https://screeb.app)                            |
+| Cost      | Freemium (_start $39/month_)                                                   |
```

Comment:
Make sure to map this to the existing Freemium examples (_$0-$XX/month_). If it doesn't start at $0, then it would be a paid product.